### DISCUSSION TITLE
Please at EEP D2-50 for Lunos Ventilation integration

### DISCUSSION THREAD

**Discussion by Torde82** [mak-gitdev/HA_enoceanmqtt#50] [Request]
Hi,

I'm new to Home Assistant and like to use the Addon to integrate my Lunos e2 Ventilation System into Home Assistant. Herefore the EEP D2-50 Profile is needed. Could you please implement it to the Addon.

Regards 
Torsten

**Comment by mak-gitdev** [comment]
Hi @Torde82,

I will have a look and tell you once it is done.

**Reply by mak-gitdev** [reply]
https://github.com/users/mak-gitdev/projects/1?pane=issue&itemId=35565661

**Comment by mak-gitdev** [comment]
Hi @Torde82, 

Done in [0.1.28](https://github.com/mak-gitdev/HA_enoceanmqtt/releases/tag/0.1.28).
Sorry for the long delay

**Reply by Torde82** [reply]
Hi,
thanks for adding after trying I see in the logs that the Lunos Sensor not requires D2-50bit A5-20-05. Could you integrate that also ? In the manual it is written D2-50 but when I install the unit it requests to be integrated as A5-20-05 (Ventilation Unit (BI-DIR)).

Thanks in advance
Torsten

**Comment by mak-gitdev** [comment]
Hi @Torde82,

Ok I will have a [look](https://github.com/users/mak-gitdev/projects/1/views/1?pane=issue&itemId=47292063).
